ldms/src/auth/ldms_auth_munge.c

```diff
@@ -174,7 +174,11 @@ int __auth_munge_xprt_begin(ldms_auth_t auth, ldms_t xprt)
 				(void*)&a->rsin, &a->sin_len);
 	if (rc)
 		return rc;
-	merr = munge_encode(&a->local_cred, a->mctx, &a->lsin, a->sin_len);
+	/* zap_rdma compatibility work-around */
```

Comment:
This comment should probably detail what we are working around a little more. I would also like to know what our approach should be to eliminating this work around in future releases, so we don't need to keep this around forever.

Reply:
@narategithub please add a bit more detail about how the original RDMA provider swapped the local/remote addresses and these addresses are passed on the wire as part of munge authentication. This comment doesn't say what we're 'working around.'

Reply:
done.